# Generated by Django 2.2 on 2020-02-23 04:36

from django.conf import settings
import django.core.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('users', '0005_auto_20200222_2210'),
    ]

    operations = [
        migrations.CreateModel(
            name='Profile',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('height', models.PositiveSmallIntegerField(null=True, validators=[django.core.validators.MinValueValidator(10), django.core.validators.MaxValueValidator(120)])),
                ('weight', models.PositiveSmallIntegerField(null=True, validators=[django.core.validators.MinValueValidator(2), django.core.validators.MaxValueValidator(1500)])),
                ('date_of_birth', models.DateField(auto_now=True)),
                ('gender', models.CharField(choices=[('M', 'Male'), ('F', 'Female'), ('O', 'Other')], max_length=1, null=True)),
                ('sleep_time', models.TimeField(null=True)),
                ('sleep_end', models.TimeField(null=True)),
                ('balance', models.PositiveIntegerField(default=0)),
                ('address', models.CharField(blank=True, max_length=200)),
                ('phone_number', models.CharField(blank=True, max_length=14)),
                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.DeleteModel(
            name='CustomUser',
        ),
        migrations.DeleteModel(
            name='FitnessChallenge',
        ),
    ]
